_sass/_components/utilities.scss

```diff
@@ -69,3 +69,54 @@
   text-indent: units(-2);
 }
 
+.height-67px {
```

Comment:
This is a reasonable temporary solution, but it's not a pattern I'd want us  to establish for future blog posts with images. Let's see if we can iterate on this before merging.

Reply:
I've got a _speculative_ fix for this in https://github.com/18F/18f.gsa.gov/pull/3863/commits/3796cb7cc8347752f266f9799b7adf7dc85a387a: I say "speculative", because I think the `post layout isn't currently getting applied to this particular blog post. I did a quick in-inspector test to make sure the layout works, but this will likely need a visual QA once the layout bug's fixed.